spec/models/organization_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Organization, type: :model do
+  describe 'associations' do
+    it 'has many tradings' do
+      association = described_class.reflect_on_association(:tradings)
+      expect(association.macro).to eq(:has_many)
+    end
+  end
+
+  describe 'validations' do
+    it 'validates presence of name' do
+      organization = Organization.new
+      expect(organization).not_to be_valid
+    end
+
+    it 'validates uniqueness of name' do
+      Organization.create(name: 'Test Organization')
```

Comment:
Can we use factoryBot to create organizations instead of direct objects?

Reply:
Resolved bhaiya. Most of the creation now use factory bot except where we need to check uniqueness